src/Plugin/views/field/OgMembershipBulkForm.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\og\Plugin\views\field;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\og\OgMembershipInterface;
+use Drupal\system\Plugin\views\field\BulkForm;
+
+/**
+ * Defines a user operations bulk form element.
+ *
+ * @ViewsField("og_membership_bulk_form")
+ */
+class OgMembershipBulkForm extends BulkForm {
+
+  /**
+   * {@inheritdoc}
+   *
+   * Provide a more useful title to improve the accessibility.
+   */
+  public function viewsForm(&$form, FormStateInterface $form_state) {
+    // Remove all actions related to (non-)member role.
+    foreach ($this->actions as $key => $action) {
+      if (strstr($key, '-member') !== FALSE) {
```

Comment:
I have fixed this in `og_og_role_insert()` by checking if the newly created role is required:

```
  // Create actions to add or remove the role, except for the required default
  // roles 'member' and 'non-member'. These cannot be added or removed.
  if ($role->getRoleType() === OgRoleInterface::ROLE_TYPE_REQUIRED) {
    return;
  }
```